frontend/src/app/app.module.ts

```diff
@@ -22,7 +22,7 @@ import { LoginPageComponent } from './pages/login-page/login-page.component';
 import { RegisterPageComponent } from './pages/register-page/register-page.component';
 
 // Providers
-import { AuthService } from './providers/auth-service/auth.service';
+import { AuthService, LobbyService } from './providers';
```

Comment:
Woowwwwwww didn't know about dis

Reply:
yeee I believe it is called a barrel index.ts file when you do it that way.